CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [4.8.1] Unreleased
+## [4.8.2] Unreleased

```

Comment:
No need for the version bump. `4.8.1` hasn't been released yet. This PR would be added to it.